packages/zk-certificates/package.json

```diff
@@ -44,16 +44,23 @@
     "circom2": "^0.2.13",
     "circomlib": "^2.0.5",
     "circomlibjs": "^0.1.7",
+    "csvtojson": "^2.0.10",
     "ethers": "^5.7.2",
     "ffjavascript": "^0.3.0",
+    "i": "^0.3.7",
```

Comment:
What is this package used for? I do not see it used in this PR.

Reply:
removed, I have no idea :D